tests/core/identity/test_identity.py

```diff
@@ -17,7 +17,7 @@ def _get_instance(self):
                 explorer_url="https://explorer.testnet.grid.tf/api/v1",
             )
         else:
-            raise Exception("Please add (name, Email, words) of your 3bot identity as Env ")
+            raise Exception("Please add (tname, email, words) of your 3bot identity as Env ")
```

Comment:
as Env means as in .env file or as environment variables? and if as environment variables, they should be all upper case